### ISSUE TITLE
parseSameOriginStylesheet tests are flakey

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2797]
This test seems to fail quite frequently. Should figure out why and fix it.

https://app.circleci.com/pipelines/github/dequelabs/axe-core/1792/workflows/3bb69f27-f8a1-422d-95fd-f650f6635ba3/jobs/27797

```sh
  axe.utils.parseSameOriginStylesheet
    ✖ returns @import rule specified in the stylesheet
```
